core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
You will have to be more explicit on what the problem is because `urlForModuleAction()` is called only once in SimplePageBuilder, and `urlForActionWithDefaultUserParams()` is called ~ once in each plugin, *sometimes* twice (and this method has a different output for each plugin). Caching `explode(get_class())` has no consequences at all unless it is called thousands of times, and here we are talking about one or two calls.